ivy_models_tests/googlenet/test_googlenet.py

```diff
@@ -0,0 +1,50 @@
+import os
+import random
+import ivy
+import pytest
+import numpy as np
+
+from ivy_models.googlenet import inceptionNet_v1
+from ivy_models_tests import helpers
+
+import jax
+
+jax.config.update("jax_enable_x64", False)
+
+load_weights = random.choice([False, True])
+model = inceptionNet_v1(pretrained=load_weights)
+v = ivy.to_numpy(model.v)
+
+
+@pytest.mark.parametrize("data_format", ["NHWC", "NCHW"])
+def test_alexnet_tiny_img_classification(device, f, fw, data_format):
+    """Test AlexNet image classification."""
```

Comment:
```suggestion
    """Test GoogleNet image classification."""
```

Just adding my two cents :)